contoso_manufacturing/developer/webapp-decode/app.py

```diff
@@ -4,47 +4,105 @@
 import json
 import numpy as np
 from yolov8 import YOLOv8OVMS
-
-
+from welding import WeldPorosity
+from pose_estimator import PoseEstimator
 
 app = Flask(__name__)
 
-OVMS_URL = "10.0.0.4:8080"
-MODEL_NAME = ''
-
 camera = None  # Initialized later based on the selected video
+latest_choice_detector = None # Global variable to keep track of the latest choice of the user
+ovms_url = "192.168.0.4:31640"
```

Comment:
Change this to be an ENV variable and deploy it as part of the helm chart - Use the name of the OVMS service (maybe change it to be nodePort service instead of loadBalancer)

Reply:
Changed